test/integration/_support/amqpPublisher.ts

```diff
@@ -10,11 +11,11 @@ export function create(topic: string, cb: Function) {
   connection.once("ready", () => {
     const publisher: BrokerPublisherAdapter = {
       publish: function () {
```

Comment:
Not really sure why this function has no parameters, but we are depending on specific `arguments` to do the  calculations below.